shared/chat/conversation/input-area/normal/container.js

```diff
@@ -180,6 +181,9 @@ const mergeProps = (stateProps, dispatchProps, ownProps: OwnProps): Props => ({
   showWalletsIcon: stateProps.showWalletsIcon,
   suggestChannels: stateProps.suggestChannels,
   suggestCommands: stateProps.suggestCommands,
+  suggestTeams: stateProps.suggestTeams
```

Comment:
This likely should be memoized. Merge probably gets called a ton especially while typing

Reply:
Yeah I was concerned about this, although I don't think typing is a factor since this input is uncontrolled.